Sources/Networking/Responses/CustomerInfoResponse.swift

```diff
@@ -74,16 +78,34 @@ extension CustomerInfoResponse {
         var productIdentifier: String
         var purchaseDate: Date?
 
+        @IgnoreEncodable @IgnoreHashable
+        var rawData: [String: Any]
```

Comment:
I think this is pretty elegant 🤓

Reply:
yeah, the syntax for this is :chefskiss: